### PR TITLE
fixed bug #21 on main codebase

### PR DESCRIPTION
**Pull request by sausheong**
Made simple change from global variable to a function that returns the struct.

### CODE REVIEW

File: image_option.go [hunk 1]
```diff
@@ -9,13 +9,15 @@ import (
...
 
 // _defaultOutputOption default output image background color and etc options
-var _defaultOutputOption = &outputImageOptions{
-	bgColor:      color.White,     // white
-	qrColor:      color.Black,     // black
-	logo:         nil,             //
-	qrWidth:      20,              //
-	shape:        _shapeRectangle, //
-	imageEncoder: jpegEncoder{},
+func _defaultOutputOption() *outputImageOptions {
```
Comment (yeqown) [inline image_option.go:12]: no need to add a underline as the function prefix

File: image_option_api_test.go [hunk 2]
```diff
@@ -8,7 +8,7 @@ import (
 )
 
 func Test_WithBuiltinImageEncoder(t *testing.T) {
-	oo := *_defaultOutputOption
```
Comment (yeqown) [inline image_option_api_test.go:11]: there is no need to keep copying semantics, just
```go
oo := defaultOutputOption()
```
this change also need to apply to the following codes

**Comment by yeqown** [comment]
@sausheong would add a testcase for `Apply` methods will not take effect to default option value?

**Comment by sausheong** [comment]
Sorry I'm not sure which ones you're talking about? I was just making a small change to make sure that it's not a global variable, all existing code still works.

**Comment by yeqown** [comment]
all right, just need you make some changes to you PR as my comment described, as for the `Apply` testcase I'll add it later.

**Comment by sausheong** [comment]
Ah I see. Thanks!

**Pull request merged**